src/cli/commands/files/cat.js

```diff
@@ -22,12 +22,22 @@ module.exports = Command.extend({
       if (err) {
         throw err
       }
+      if (utils.isDaemonOn()) {
+        ipfs.cat(path, (err, res) => {
+          if (err) {
+            throw err
+          }
+          console.log(res.toString())
+        })
+        return
+      }
       ipfs.files.cat(path, (err, res) => {
         if (err) {
           throw (err)
         }
+        console.log(res)
         if (res) {
-          res.on('file', (data) => {
+          res.on('data', (data) => {
             data.stream.pipe(process.stdout)
           })
```

Comment:
I believe this console.log should not be here and the if (res) is not necessary.